Dockerfile

```diff
@@ -4,4 +4,8 @@ RUN apt-get update
 RUN apt-get install -y git
 
 RUN yarn global add github:stanford-oval/almond-cloud
-RUN yarn
\ No newline at end of file
+RUN yarn
+
+RUN echo $DATABASE_URL
+RUN almond-cloud bootstrap
```

Comment:
You cannot do this, this would try to bootstrap in the build environment, which doesn't make sense.